spec/amp-var-substitutions.md

```diff
@@ -478,6 +479,17 @@ Use `COUNTER(name)` to generate an index for the given request. Counts start at
 * **platform variable**: `COUNTER`
 * **amp-analytics variable**: `${counter}`
 
+#### Sum
+
+Use `SUM(name, value)` to generate a sum for the given request. Values are added per a given `name`.
+
+* **platform variable**: `SUM`
+*  Example: <br>
+  ```html
+  <amp-pixel src="https://foo.com/pixel?SUM('pixel1', 5)"></amp-pixel>
```

Comment:
to follow the convention, there should not be the single quote `'` around string literals.

Reply:
Done.